### DISCUSSION TITLE
How to implement file upload?

### DISCUSSION THREAD

**Discussion by andrashatvani** [ExpediaGroup/graphql-kotlin#1037] [Q&A] [answered]
Currently there's no example demonstrating the implementation of a file upload. Desired would be one example with both, single and multiple files.

**Comment by smyrick** [comment, marked as answer]
From the issue https://github.com/ExpediaGroup/graphql-kotlin/issues/1036

> Apollo file uploads are not part of the GraphQL specification and are not supported "out-of-the-box". As per #518 you will have to
> configure Jackson object mapper to correctly handle those types (and potentially also configure custom data fetcher to apply the
> custom logic - if needed).
> There are some examples linked in other related issues #661 (comment) and #518 (comment)
